### PR TITLE
Adding @sengrid/mail definition

### PR DESCRIPTION
**Pull request by rimiti**
Hello everybody,

I've added the `@sendgrid/mail` definition.

Have a nice day.
Dimitri,

### CODE REVIEW

File: definitions/npm/@sendgrid/mail_v6.x.x/flow_v0.30.x-/test_mail_v6.x.x.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import mail from '@sendgrid/mail';
+
+mail.setApiKey('myApiKey');
+mail.send({
+  from: {email: "test@example.com"},
+});
```
Comment (GAntoine) [inline definitions/npm/@sendgrid/mail_v6.x.x/flow_v0.30.x-/test_mail_v6.x.x.js]: You'll need to add some failing assertions before we can merge this in. Could you also add just a couple more tests for the required keys in `MailData` and `MailContent`, among the others above?

**Comment by rimiti** [comment]
@GAntoine you'll be glad to see that I've added lots of tests.

**Comment by GAntoine** [comment]
Awesome! I went ahead and added some failing statements wrapped in `// $ExpectError`.

**Comment by rimiti** [comment]
It's a good team work ! Thank you 👍 🥇

**Pull request merged**